src/CitMovie.Data/FollowRepository/FollowRepository.cs

```diff
@@ -12,36 +13,33 @@ public FollowRepository(FrameworkContext context)
             _context = context;
         }
 
-        public async Task<IList<Follow>> GetFollowings(int userId, int page, int pageSize)
+        public async Task<IList<FollowDto>> GetFollowingsAsync(int userId, int page, int pageSize)
         {
             return await _context.Follows
                 .Where(f => f.UserId == userId)
                 .Skip(page * pageSize)
                 .Take(pageSize)
+                .Select(f => new FollowDto
+                {
+                    FollowingId = f.FollowingId,
+                    PersonId = f.PersonId,
+                    FollowedSince = f.FollowedSince
```

Comment:
This should not happen In the repository but in the business service. The repository should essentially never know about the DTO and the Api/Webservice should never know about the domain object models

Reply:
Actually lets just merge the PR, then we can go back and use AutoMapper for converting Domain Models to DTOs in another PR